src/features/Post/components/PostItem/index.jsx

```diff
@@ -45,12 +52,17 @@ const PostItem = ({ post }) => {
 		getIsPostLikedByAuthUser()
 	);
 
+	const getIsPostInBookmarks = () => bookmarks.includes(_id);
+
+	const [isPostInBookmarks, setIsPostInBookmarks] = useState(
+		getIsPostInBookmarks()
```

Comment:
check this out and [Avoid recreating the initial state](https://beta.reactjs.org/apis/usestate#avoiding-recreating-the-initial-state)

Reply:
Yes, I have made this change 👍🏽 
Thanks!